docs/resources/modelarts_dataset.md

```diff
@@ -0,0 +1,195 @@
+ Default value is `true`. Changing this parameter will create a new resource.
+
+* `label_format` - (Optional, String, ForceNew) Specifies the custom format information of labeled files when import
+ labeled files for Text classification. Structure is documented below.
+ Changing this parameter will create a new resource.
+
+* `schemas` - (Optional, List, ForceNew) Specifies the schema information of source data when `type` is `400`.
+ Structure is documented below. Changing this parameter will create a new resource.
+
+* `labels` - (Optional, List) Specifies labels information. Structure is documented below.
+
+The `data_source` block supports:
+
+* `data_type` - (Required, Int, ForceNew) Specifies the type of data source. The options are as follows:
+  + *0*: OBS.
+  + *1*: GaussDB(DWS).
+  + *2*: DLI.
+  + *4*: MRS.
+  
+ Changing this parameter will create a new resource.
+
+* `path` - (Optional, String, ForceNew) Specifies the OBS path when `data_type` is `0`
+ or the hdsf path when `data_type` is `4`. All the file in this directory and subdirectories will be which be imported
+ to the dataset. Changing this parameter will create a new resource.
+
+* `with_column_header` - (Optional, Bool, ForceNew) Specifies whether the data contains table header when the type
+ of dataset is `400`(Table type). Default value is `true`. Changing this parameter will create a new resource.
+
+* `queue_name` - (Optional, String, ForceNew) Specifies the queue name of DLI when `data_type` is `2`.
+ Changing this parameter will create a new resource.
+
+* `database_name` - (Optional, String, ForceNew) Specifies the database name of DWS/DLI when `data_type` is `1` or `2`.
+ Changing this parameter will create a new resource.
+
+* `table_name` - (Optional, String, ForceNew) Specifies the table name of DWS/DLI when `data_type` is `1` or `2`.
+ Changing this parameter will create a new resource.
+
+* `cluster_id` - (Optional, String, ForceNew) Specifies the cluster ID of DWS/MRS when `data_type` is `1` or `4`.
+ Changing this parameter will create a new resource.
+
+* `user_name` - (Optional, String, ForceNew) Specifies the user name of databse when `data_type` is `1`.
+ Changing this parameter will create a new resource.
+
+* `password` - (Optional, String, ForceNew) Specifies the password of database when `data_type` is `1`.
+ Changing this parameter will create a new resource.
+
+The `schemas` block supports:
+
+* `type` - (Required, String, ForceNew) Specifies the field type. Valid values include: `String`, `Short`, `Int`,
+ `Long`, `Double`, `Float`, `Byte`, `Date`, `Timestamp`, `Bool`. Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the field name. Changing this parameter will create a new resource.
+
+The `label_format` block supports:
+
+* `type` - (Required, Int, ForceNew) Specifies Label type for text classification.
```

Comment:
The type in the schema is `String` and the behavior is not `Required`.

Reply:
fix